tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Cpp.cs

```diff
@@ -73,15 +77,122 @@ public class Cpp : LanguageBase
+            }
+
+            foreach (var package in packageVersions.Keys)
+            {
+                var packageVersion = packageVersions[package];
+                // we don't need to make any updates we want the latest version
+                if (String.Compare(packageVersion, "source", true) == 0)
+                {
+                    continue;
+                }
+                bool found = false;
+
+                foreach (VcpkgDependency dependency in document.Dependencies)
+                {
+                    if (dependency.Name == package)
+                    {
+                        dependency.VersionGt = packageVersion;
+                        found = true;
+                        break;
+                    }
+                }
+
+                if (!found)
+                {
+                    VcpkgDependency vcpkgDependency = new VcpkgDependency();
+                    vcpkgDependency.Name = package;
+                    vcpkgDependency.VersionGt = packageVersion;
+                    document.Dependencies.Add(vcpkgDependency);
+                }
+
+                if (document.Overrides == null)
+                {
+                    document.Overrides = new List<VcpkgDependency>();
+                }
+
+                found = false;
+
+                foreach (VcpkgDependency overrideEntry in document.Overrides)
+                {
+                    if (overrideEntry.Name == package)
+                    {
+                        overrideEntry.Version = packageVersion;
+                        found = true;
+                        break;
+                    }
+                }
+
+                if (!found)
+                {
+                    VcpkgDependency overrideEntry = new VcpkgDependency();
+                    overrideEntry.Name = package;
+                    overrideEntry.Version = packageVersion;
+                    document.Overrides.Add(overrideEntry);
+                }
+                updated = true;
+            }
+
+            if (updated)
+            {
+                using (StreamWriter writer = new StreamWriter(fullVcpkgPath))
+                {
+                    string serializedDocument = JsonConvert.SerializeObject(document, Formatting.Indented);
+                    Console.WriteLine($"Updated {fullVcpkgPath}{Environment.NewLine}{serializedDocument}");
+                    writer.Write(serializedDocument);
+                }
+            }
+
+            return updated;
+        }
+
+        private string ComposeGitTag(string project, string packageVersion)
+        {
+            StringBuilder tag = new StringBuilder(project.Length + packageVersion.Length);
+
+            string[] parts = project.Split('-', StringSplitOptions.RemoveEmptyEntries);
+            tag.Append(String.Join('-', parts.Take(parts.Length - 1).ToArray<string>()));
+            tag.Append('_');
+            tag.Append(packageVersion);
+
+            return tag.ToString();
```

Comment:
```suggestion
```

Looks like this code is now unused and can be removed

Reply:
done

Reply:
I am still seeing this code in the PR.  Did you intend to push a change that removes this?